### PR TITLE
setup fixes

### PR DESCRIPTION
**Pull request by desbo**
A couple of tweaks to the setup script and a troubleshooting step for an error I had today.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,7 @@ jspm registry config github
 It'll ask for a GitHub access token. Go to GitHub Settings -> Applications and [generate new token](https://github.com/settings/tokens/new). Ensure only the public_repo scope is checked.
 Now create a registry instance.
 ```
+sudo npm -g install jspm-bower-endpoint # jspm >= 0.15.0
```
Comment (sndrs) [inline README.md]: `sudo`?

File: setup.sh [hunk 2]
```diff
@@ -101,10 +101,11 @@ install_grunt() {
 install_jspm() {
   if ! installed jspm; then
     sudo npm -g install jspm
-    jspm registry config github
-    jspm registry create bower jspm-bower-endpoint
   fi
-  jspm install
+ 
+ sudo npm -g install jspm-bower-endpoint
```
Comment (sndrs) [inline setup.sh]: `sudo` shouldn't be necessary

Comment (desbo) [inline setup.sh]: i think it is for global?

Comment (piuccio) [inline setup.sh]: you can install globally without `sudo`

**Pull request merged**